docs/features/content-extensions/contentManager.md

```diff
@@ -0,0 +1,40 @@
+---
+id: contentManager
+title: ContentManager
+sidebar_label: ContentManager
+description: Extensions for the MonoGame ContentManager class.
+---
+
+:::tip[Up to date]
+This page is **up to date** for MonoGame.Extended `@mgeversion@`.  If you find outdated information, [please open an issue](https://github.com/craftworkgames/craftworkgames.github.io/issues).
+:::
+
+# ContentManager extensions
+
+## ContentManager.OpenStream
+
+`System.IO.Stream ContentManager.OpenStream(string filename)`
+
+**OpenStream** allows easy access to `TitleContainer.OpenStream` so you can use the `Game.Content` object to load compiled resources *and* included resources.
```

Comment:
This may need some rewording.  
1. `OpenStream` will load `.xnb` files (compiled) assets, but not in the same way as loading them as a concrete type.  For instance, you could use it to open *image.png* and then use the stream returned to create a `Texture2D` without more steps.

2. Can probably drop the mention of `TitleContainer.OpenStream`, even though this is what it uses under the hood.  It still relies on the `Content.RootDirectory` value set in the path that's generated.  So, the `path` parameter needs to be relative to `Content.RootDirectory` and not `TitleContainer`'s base path.

Reply:
Ok, I've made edits in hopes it aligns with what you are asking.  Let me know